src/components/GameView/GameMenu.vue

```diff
@@ -113,8 +110,20 @@ export default {
         ? 'The game will end and your opponent will win.'
         : 'If your opponent agrees, the game will end in a stalemate. The request will cancel if the opponent declines or either player makes a move';
     },
+    buttonSize() {
+      return this.$vuetify.display.mdAndDown ? 'small' : 'medium';
+    },
   },
   methods: {
+    closeMenu() {
+      this.showGameMenu = false;
+    },
+    openRulesDialog() {
+      this.showRulesDialog = true;
+      this.showConcedeDialog = false;
+      this.showStalemateDialog = false;
+      this.showGameMenu = false;
```

Comment:
This makes sense and is consistent with how we have the other dialogs, but I don't love the overlap between the various opening and closing functions, given that the dialogs are mutually exclusive and we have to close all the other ones every time we open something.

I wonder if it would make sense to use a single string-values variable for which dialog is open and use computed props to consume it for each individual dialog. Something like:

```
data() {
  return {
    shownDialog: ''; // '' | 'rules' | 'concede' | 'stalemate'
  }
},
computed: {
  showRuleaDialog: {
    get: () => this.shownDialog === 'rules',
    set(val) {
      this.shownDialog = val ? 'rules' : '';
    }
  },
  // one computed prop like above per dialog
}

```
Then each of the dialogs could v-model its respective computed prop. I think we might also want to in-line the @ click handlers to assign to the variable in the html at that point rather than defining a specific method for opening each dialog, opening a dialog would only require assigning a value to one variable.

This might actually be more total lines of code so idk if it's better but the idea is that when opening a dialog we could just assign a value to the one piece of data like `this.shownDialog = 'rules';` and that would take care of turning all the others off because they all read from the same variable. @Haviles04 what do you think?

Reply:
I like this, I don't think we need the setter in the computed value, we can just do `return this.shownDialog === 'rules' 
and set shownDialog inline like `shownDialog = 'rules'` in the html

Reply:
The only funky thing about this is that we're using `v-model` which is a 2-way binding that is literally shorthand for 

```
:modelValue="shownDialog"
@update:modelValue="shownDialog = $event"
```

Any of these components trying to close themselves by emitting an `update:modelValue` events, it will be ignored and throw an error because `shownDialog` is a computed prop without a setter